mender-convert

```diff
@@ -440,9 +440,9 @@ do_make_sdimg_qemux86_64() {
 
   log "\tExtracting root filesystem partition from raw disk image."
   extract_file_from_image ${raw_disk_image} ${rootfs_part_start} \
-                          ${rootfs_part_size} "rootfs.img"
+                          ${rootfs_part_size} ${mender_rootfs_basename}
```

Comment:
@mirzak Isn't it misleading to extract raw disk root filesystem to a file named, e.g. for raspberrypi3 `output/mender-raspberrypi3-release-1_1.5.ext4`? Besides you do mentioned that not always raw disk's rootfs partition must be of ext4 type. Wouldn't it be "safer" to just call it `rootfs.img` as it was before or add a new pair of global variables: `raw_rootfs_image`, `raw_rootfs_basename`. Moreover during artifact creation you once again write with **dd** command to that disk image.

The same refers to the _mender-convert_ file in the next commit.